src/main/java/lotto/domain/RankResult.java

```diff
@@ -1,24 +1,26 @@
 package lotto.domain;
 
-import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
+import java.util.stream.Collectors;
 
 public class RankResult {
 
-    private static final int INITIAL_VALUE = 0;
-    private final Lottos lottos;
-    private final Winning winning;
-    private Map<Rank, Integer> rankResult;
+    private static int DEFAULT_NUMBER = 0;
+    private static int AUTO_INCREMENT = 1;
```

Comment:
```suggestion
    private static final int DEFAULT_NUMBER = 0;
    private static final int AUTO_INCREMENT = 1;
```
`final` 키워드가 빠졌네요 ″̮